flytekit/core/array_node_map_task.py

```diff
@@ -347,6 +349,41 @@ def _raw_execute(self, **kwargs) -> Any:
 
 
 def map_task(
+    target: Union[LaunchPlan, PythonFunctionTask],
+    concurrency: Optional[int] = None,
+    min_successes: Optional[int] = None,
+    min_success_ratio: float = 1.0,
+    **kwargs,
+):
+    """
+    Wrapper that creates a map task utilizing either the existing ArrayNodeMapTask
+    or the drop in replacement ArrayNode implementation
```

Comment:
Can you add a newline after the description? This is causing the monodocs [failure](https://github.com/flyteorg/flytekit/actions/runs/10155606942/job/28084523310?pr=2480) we're seeing in CI. The error is not intuitive, linking the relevant portion of it:
```
2024-07-30T04:55:03.9312901Z sphinx.errors.SphinxWarning: /home/runner/work/flytekit/flytekit/flyte/docs/_src/flytekit/flytekit/core/array_node_map_task.py:docstring of flytekit.core.array_node_map_task.map_task:8:Block quote ends without a blank line; unexpected unindent.
```